Source/WebCore/page/Quirks.cpp

```diff
@@ -892,6 +893,13 @@ bool Quirks::shouldBypassBackForwardCache() const
             return documentLoader->response().cacheControlContainsNoStore();
     }
 
+    // Spinner issue from image search for bing.com.
+    if (registrableDomain == "bing.com"_s) {
+        static MainThreadNeverDestroyed<const AtomString> imageSearchDialogID("sb_sbidialog"_s);
+        if (RefPtr element = document->getElementById(imageSearchDialogID.get()))
```

Comment:
nit: I don't think you need a `RefPtr` here since `Node::renderer()` looks like a trivial function. Could likely just use `auto*`